.snippets/text/disclaimers/educational-tutorial.md

```diff
@@ -0,0 +1 @@
+This tutorial is for educational purposes only. As such, any contracts or code created in this tutorial should not be used in production.
```

Comment:
@wuxinxin2000 can you please translate this?

Reply:
Done.